numpy/core/src/multiarray/einsum.dispatch.c.src

```diff
@@ -1,47 +1,26 @@
 /*
- * This file provides optimized sum of product implementations used internally
- * by einsum.
```

Comment:
I think the old comment was better, this does not implement a function named `einsum`.